bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -325,6 +335,12 @@ private boolean areWeLoggedIn() throws ExecutionException, InterruptedException,
     private ContentResponse postVerisureAPI(String url, String data, boolean isJSON)
             throws ExecutionException, InterruptedException, TimeoutException {
         logger.debug("postVerisureAPI URL: {} Data:{}", url, data);
+        /*
+         * Request request = httpClient.newRequest(url).method(HttpMethod.OPTIONS);
+         * ContentResponse optionsRsp = request.send();
+         * int httpRspCode = optionsRsp.getStatus();
+         */
```

Comment:
I guess this comment has to be suppressed.

Reply:
Removed code!